tests/components/smartthings/test_climate.py

```diff
@@ -304,10 +301,8 @@ def air_conditioner_fixture(device_factory):
     await setup_platform(hass, CLIMATE_DOMAIN, devices=[air_conditioner])
     state = hass.states.get("climate.air_conditioner")
     assert state.state == HVAC_MODE_HEAT_COOL
-    assert (
-        state.attributes[ATTR_SUPPORTED_FEATURES]
-        == SUPPORT_FAN_MODE | SUPPORT_TARGET_TEMPERATURE
-    )
+    assert (state.attributes[ATTR_SUPPORTED_FEATURES] == SUPPORT_FAN_MODE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)